bindings/cpp/newapi/result_entry.cpp

```diff
@@ -35,18 +35,28 @@ dnet_record_info lookup_result_entry::record_info() const {
 
 	deserialize(raw_data(), response);
 
+	auto copy_checksum = [](const std::vector<unsigned char> &src, unsigned char *dst) {
+		if (src.size()) {
+			memcpy(dst, src.data(), DNET_CSUM_SIZE);
+		} else {
+			memset(dst, DNET_CSUM_SIZE, 0);
```

Comment:
```suggestion
			memset(dst, 0, DNET_CSUM_SIZE);
```

Reply:
Wow. Wonderful: I've searched for documentation before use, but possibly cross-eye occured =)
Found and fixed all memsets in PR (not pushed yet).